README.md

```diff
@@ -12,19 +12,18 @@ This application is a denormalised warehouse for multiple LIMS.
 1. Install the relevant ruby is installed - have a look in the `.ruby-version` file
 1. Install bundler the version of bundler used to create the `Gemfile.lock`:
 
-       gem install bundler -v $(tail -1 Gemfile.lock)
+       `gem install bundler -v $(tail -1 Gemfile.lock)`
 
-1. Install the relevant gems using bundler:
+1. Run the setup process:
 
-       bundle install
+       `bin/setup`
 
```

Comment:
I've been updating the bin/setup scripts as I use them, makes on-boarding developers a little easier.